src/phone-input/country-select.js

```diff
@@ -21,7 +21,7 @@ import type {CountryT, CountrySelectPropsT} from './types.js';
 
 CountrySelect.defaultProps = {
   disabled: defaultProps.disabled,
-  inputRef: null,
```

Comment:
I believe if we cast this object to the `$Shape<PropsT>` this bug would have been caught earlier. ex:
```js
CountrySelect.defaultProps = ({
  disabled: defaultProps.disabled,
  inputRef: null,
  maxDropdownHeight: defaultProps.maxDropdownHeight,
  maxDropdownWidth: defaultProps.maxDropdownWidth,
  overrides: {},
  size: defaultProps.size,
}: $Shape<CountrySelectPropsT>);
```

Reply:
I added this explicit casting for all the other components in phone-input. It brought up a couple other places that were misaligned so I am including the fixes in this PR.